src/main/kotlin/org/rust/cargo/commands/Cargo.kt

```diff
@@ -53,6 +55,7 @@ class Cargo(
             .withParameters(command)
             .withParameters(additionalArguments)
             .withEnvironment(environmentVariables)
+            .withEnvironment(CargoConstants.RUSTC_ENV_VAR, pathToRustExecutable)
```

Comment:
Will change, how do you usually do this, with a separate commit or a force push to the branch?